Components/Decks/DraftDeckEditor.jsx

```diff
@@ -80,7 +80,7 @@ class DraftDeckEditor extends React.Component {
         return remainingCards;
     }
 
-    getDeckFromState(restrictedList) {
```

Comment:
There's still an existing call to this method passing restrictedList

Reply:
fixed